app/scripts/modules/core/src/instance/loadBalancer/InstanceLoadBalancerHealth.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+
+export interface InstanceLoadBalancer {
+  healthState?: string;
```

Comment:
are all of these really optional?

Reply:
The way the old code relied on it, I couldn't tell if anything was strictly required because there were fallbacks.

When the primary candidate is available, the fallbacks seem optional? Is there a way in TS to say "at least one of these"?

Reply:
Yes, but it's ugly. Fine as is!